app/assets/javascripts/initializers/initializeRuntimeBanner.js

```diff
@@ -0,0 +1,60 @@
+/* global Runtime */
+
+function handleDismissRuntimeBanner(event) {
+  const runtimeBanner = document.querySelectorAll('.runtime-banner')[0];
+  if (runtimeBanner) {
+    runtimeBanner.remove();
+  }
+}
+
+// A separate function is needed for this because it's impossible to test a
+// custom scheme launch in Cypress. More about this here:
+// https://medium.com/cypress-io-thailand/understand-stub-spy-apis-that-will-make-your-test-better-than-ever-on-cypress-io-797cb9eb205a
+function launchCustomSchemeDeepLink(targetURL) {
+  window.location.href = targetURL;
+}
+
+function initializeRuntimeBanner() {
+  // This will provide the dismiss functionality for the Runtime Banner
+  const bannerDismiss = document.querySelectorAll(
```

Comment:
Random question: if we only need the first element, can't we use `querySelector` instead of `querySelector(...)[0]`?